test/unit/Query/QueryCollection.test.php

```diff
@@ -4,24 +4,71 @@
 use Gt\Database\Connection\DefaultSettings;
 use Gt\Database\Connection\Driver;
 use Gt\Database\Result\ResultSet;
+use PHPUnit_Framework_MockObject_MockObject;
 
 class QueryCollectionTest extends \PHPUnit_Framework_TestCase {
+/** @var  QueryCollection */
+private $SUT;
```

Comment:
Hmm, after a quick search online it looks pretty standard terminology, but I didn't know what it meant. I'll let you decide what's best between SUT or queryCollection, so change if you fancy it. 

I'll merge this before lunch.